src/app/user-edit/user-edit.component.html

```diff
@@ -6,8 +6,30 @@
         <form class="UserEdit">
             <p><span class="bolded">Name: </span> <input type="text" [(ngModel)]="userEdit.name" name="name" /></p>
             <p><span class="bolded">Jam ZIP Code: </span> <input type="text" [(ngModel)]="userEdit.zipcode" name="zipcode" /></p>
-            <p><span class="bolded">My Instruments: </span> <input type="text" [(ngModel)]="userEdit.instruments" name="instruments"/></p>
-            <p><span class="bolded">Genre: </span><input type="text"  [(ngModel)]="userEdit.genres" name="genre"/></p>
+            <p><span class="bolded">My Instruments: </span> <select [(ngModel)]="userEdit.instrument" name="instrument">
+                <option>Guitar</option>
+                <option>Piano</option>
+                <option>Drums</option>
+                <option>Flute</option>
+                <option>Clarinet</option>
+                <option>Bass</option>
+                <option>Triangle</option>
+                <option>Cowbell</option>
+                <option>Theremin</option>
+                <option>Saxophone</option>
+            </select></p>
+            <p><span class="bolded">Genre: </span><select  [(ngModel)]="userEdit.genre" name="genre">
+                <option>Pop</option>
```

Comment:
May consider iterating over an array of static data - then you only need to update the array and `options` will be populated dynamically - same for above list

Reply:
Ooh, that's interesting, I'll ask the team if they think it's feasible to add this before the code freeze.